src/sentry/static/sentry/app/views/organizationDashboard/widget.jsx

```diff
@@ -80,6 +74,10 @@ class Widget extends React.Component {
 export default withRouter(withOrganization(withGlobalSelection(Widget)));
 export {Widget};
 
+const StyledPanel = styled(Panel)`
+  margin-bottom: ${space(2)};
+`;
+
 const StyledPanelBody = styled(PanelBody)`
   height: 200px;
```

Comment:
Currently we only have 1 widget that would really benefit from a dynamic height, we can circle back to this in the future.